apis/v1alpha2/httproute_types.go

```diff
@@ -523,8 +523,7 @@ type HTTPRouteFilter struct {
 	// +optional
 	RequestRedirect *HTTPRequestRedirectFilter `json:"requestRedirect,omitempty"`
 
-	// URLRewrite defines a schema for a filter that responds to the
-	// request with an HTTP redirection.
+	// URLRewrite defines a schema for a filter that rewrites an HTTP request's URL.
```

Comment:
Let's use the same language as the full description comment:

`RequestRedirect defines a filter that modifies a request during forwarding.`

Reply:
i don't get it. should i change the comment in URLRewrite to "URLRewrite defines a filter that modifies a request during forwarding." ? when mean "the same language as the full description comment"?